CHANGELOG.asciidoc

```diff
@@ -40,6 +40,8 @@ https://github.com/elastic/beats/compare/v6.0.0-beta1...master[Check the HEAD di
 - Register kubernetes `field_format` matcher and remove logger in `Encode` API {pull}4888[4888]
 - Fix go plugins not loaded when beat starts {pull}4799[4799]
 
```

Comment:
Empty line here probably coming from rebase?

Reply:
Yeah, i would ignore that for now, I'm doing reformatting when closing the the CHANGELOG anyway.